avogadro/mainwindow.cpp

```diff
@@ -1496,7 +1502,7 @@ void MainWindow::buildMenu()
 
 #ifdef Q_OS_MAC
   // hide the file toolbar on Mac
-  m_fileToolBar->hide();
+//  m_fileToolBar->hide();
```

Comment:
(shrug)

The bug occurs at least on 10.12 with Qt - when the toolbar is hidden, it still shows a weird white box.

The other part of the patch avoids adding the toolbar in the first place, so it doesn't need to be hidden.

I'm fine with taking that code out, but there are some advantages for history (e.g., if the Qt bug is fixed, presumably it can be hidden again).